htmd/builder/amber.py

```diff
@@ -325,21 +325,37 @@ def build(mol, ff=None, topo=None, param=None, prefix='structure', outdir='./bui
         f.write('loadamberprep {}\n'.format(newname))
     f.write('\n')
 
-    # Detect disulfide bridges if not defined by user
-    if disulfide is None and not ionize:
-        logger.info('Detecting disulfide bonds.')
-        disulfide = detectDisulfideBonds(mol)
-
-    # Fix structure to match the disulfide patching
-    if not ionize and len(disulfide) != 0:
+    # TODO: Remove this once we deprecate the class
+    from htmd.builder.builder import DisulfideBridge
+    from htmd.molecule.molecule import UniqueResidueID
+    if disulfide is not None and len(disulfide) != 0 and isinstance(disulfide[0], DisulfideBridge):
```

Comment:
Maybe additional deprecation warning? (even though it is on the correct place on the `DisulfideBridge` class)

Reply:
I prefer to only throw one warning instead of two. It is not possible to come to this condition without facing the first warning.

Reply:
You're right.